supports-color/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for supports-color 3.1.2
+// Project: https://github.com/chalk/supports-color
+// Definitions by: Melvin Groenhoff <https://github.com/mgroenhoff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare interface SupportsColor {
+    level: number;
+    hasBasic: boolean;
+    has256: boolean;
+    has16m: boolean;
+}
+
+declare const supportsColor: boolean & SupportsColor;
```

Comment:
Looking at the source, looks like it's `SupportsColor | false`.
(Although the last line `module.exports = process && support(supportLevel);` implies that it will be undefined if `process` is, but `process` is used earlier anyway, so loading the module would fail if `process` were undefined...)